framework/core/Vcs.pm

```diff
@@ -115,15 +115,15 @@ A Vcs object has to be instantiated with:
 
 =item * Repository url
 
-=item * File name of the commit database (commit-db), see below for details
+=item * File name of the commit database (active-bugs csv), see below for details
 
 =item * Reference to post-checkout hook (optional) -- if provided, this method is called after each checkout.
 
 =back
 
-=head2 commit-db
+=head2 active-bugs csv
 
-The commit-db (csv) file has the structure: C<bug_id,revision_buggy,revision_fixed>.
+The active-bugs (csv) file has the structure: C<bug_id,revision_buggy,revision_fixed>.
```

Comment:
This documentation is outdated. There are more fields in this file.

Reply:
Updated